Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:6-stretch
+LABEL maintainer="will.price94@gmail.com"
+LABEL version="0.0.1"
+RUN apt-get update -qq && \
+    apt-get install -y python
+# Prevent npm from spamming
+RUN npm config set loglevel=warn && \
+    npm config set progress=false
+WORKDIR /app/
+COPY package.json package-lock.json /app/
+RUN npm install
+COPY . /app/
+RUN npm run build
+
+WORKDIR /app
+VOLUME /app/db
+EXPOSE 3000
+ENV VIMFLOWY_PASSWORD=vimflowy123
```

Comment:
I think empty is a better default than a fixed string

Reply:
That's not possible, environment variables in dockerfiles have to have a value (see [ENV](https://docs.docker.com/engine/reference/builder/#env))

Reply:
can it be empty string?

Reply:
Not sure, I'll check. Is vimflowy happy if we launch with `--password ''`?

Reply:
should be!

Reply:
We can't have an empty string, docker doesn't support them in Dockerfiles.